lib/cog/commands/help/command_formatter.ex

```diff
@@ -1,74 +1,34 @@
 defmodule Cog.Commands.Help.CommandFormatter do
```

Comment:
Yeah I guess this is leftover design from the string formatting. It's tough because we're really only using some of this stuff, for instance the synopsis and option string generation, in the help template. It doesn't seem generic enough for a view, but I can see the argument when looking at just the subcommands json rendering for instance. Let me think about this for a bit.